libs/components/datetime/src/lib/modules/date-range-picker/types/date-range-calculator.ts

```diff
@@ -13,7 +13,12 @@ export class SkyDateRangeCalculator {
   /**
    * The text to display in the calculator select menu.
    */
-  public readonly shortDescription: string;
+  public shortDescription: string;
```

Comment:
Why is this no longer `readonly`?

Reply:
We're creating the calculators on initialization now, so I needed a way to set the `shortDescription` after the resources strings had been resolved. You're correct, though, this alters the public API so I went about it a bit differently. I created a story to rip out all the service's async stuff in version 11.